transform/app/api/routes/transform/batch.py

```diff
@@ -33,9 +34,20 @@ async def batch_update(
     data: dict | list[dict],
 ):
     """Transform a batch of a single type of data. Used for live update"""
+    task_ids = []
```

Comment:
I believe it could be just a dict with key-value pairs. 
There may be more than 1 task, but still it is about a single event